src/bootstrap/dist.rs

```diff
@@ -990,12 +989,12 @@ impl Step for PlainSourceTarball {
                    .arg("--debug")
                    .arg("--vers").arg(CARGO_VENDOR_VERSION)
                    .arg("cargo-vendor");
-                if let Some(dir) = builder.openssl_install_dir(builder.config.build) {
-                    builder.ensure(native::Openssl {
-                        target: builder.config.build,
-                    });
-                    cmd.env("OPENSSL_DIR", dir);
-                }
+                // if let Some(dir) = builder.openssl_install_dir(builder.config.build) {
```

Comment:
Hm, I think we probably want to keep this? Or I guess maybe just delete it since we're not managing openssl on our self?

Reply:
Oops I meant to delete this! I built OpenSSL in at least one of our containers so I'm hoping that this isn't necessary any more, but if it fails on the bots I'll probably need to find a workaround